modules/overview-of-openshift-gitops.adoc

```diff
@@ -0,0 +1,39 @@
+[id="overview-of-openshift-gitops"]
+= Overview of OpenShift GitOps
+include::modules/common-attributes.adoc[]
+include::modules/gitops-document-attributes.adoc[]
+:context: overview-of-openshift-gitops
+toc::[]
+
+:FeatureName: OpenShift GitOps
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+Use Red Hat OpenShift GitOps to create repeatable processes for managing OpenShift Container Platform clusters and applications across multi-cluster Kubernetes environments. When you lose time in deployment and release cycles, Red Hat Openshift GitOps is a tool that handles and automates the fast pace and complex deployments. The Red Hat Openshift GitOps’ workflow pushes from application tp development environment to testing environment to staging to production.
```

Comment:
Suggest adding this sentence here: It either deploys a new application or updates an existing one, so that you only need to update the repository.  Red Hat Openshift GitOps handles and automates everything else.